src/histolab/tiler.py

```diff
@@ -519,7 +537,7 @@ class ScoreTiler(GridTiler):
 
     def __init__(
         self,
-        scorer: Scorer,
+        scorer: Union[NucleiScorer, RandomScorer],
```

Comment:
Why having the concrete implementations here instead of the abstract one? Isn't the abstract implementations there for this purpose?

Reply:
Actually Scorer is only used in the scorer module and so it has to be private, and here we cannot import a private class, and BTW this way is more explicit for now

Reply:
To me, Scorer shouldn't be private. It has been created to define the interface (it is a protocol) and to be used as an abstract class for the user too (if one wants to implement their own concrete Scorer) and still be compliant with the type hints

some refs:
https://stackoverflow.com/questions/38791739/function-annotation-for-subclasses-of-abstract-class
https://github.com/python/mypy/pull/2853

Reply:
ok gotcha